pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -68,6 +75,8 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+            - mountPath: /var/lib/cephcsi/log/
+              name: csi-log
```

Comment:
this is mounted to csi-resizer container but i don't see any flag updated in csi-resizer to write log to a file, its same other few other containers as well.

Reply:
removed from csi-resizer, 
And rest of the containers where I didn't added it because it doesn't have support to  klog

Reply:
resizer/csi-addons/csi-omap-generator/liveness doesn't support writing log to a file? for csiaddons please open up an issue with csi-addons to provide the support for it. can you please check we are doing this for all the containers in the csi deployment/daemonset

Reply:
It just `resizer` which doesnt support it

Reply:
can you please recheck this one, resizer also uses the klog

Reply:
```
flag provided but not defined: -alsologtostderr
Usage of /csi-resizer:
  -csi-address string
    	Address of the CSI driver socket. (default "/run/csi/socket")
  -feature-gates value
    	A set of key=value paris that describe feature gates for alpha/experimental features for csi external resizer.Options are:
    	AllAlpha=true|false (ALPHA - default=false)
    	AllBeta=true|false (BETA - default=false)
    	AnnotateFsResize=true|false (ALPHA - default=false)
    	RecoverVolumeExpansionFailure=true|false (ALPHA - default=false)
    	VolumeAttributesClass=true|false (ALPHA - default=false)
  -handle-volume-inuse-error
    	Flag to turn on/off capability to handle volume in use error in resizer controller. Defaults to true if not set. (default true)
  -http-endpoint :8080

```

Reply:
sorry i was testing old resizer, it looks like a missing functionality in resizer.

Reply:
can you please open a issue in external-resizer to add support to it?

Reply:
Something wrong on how they have implemented https://github.com/search?q=repo%3Akubernetes-csi%2Fexternal-resizer+klog.&type=code

`klog.InitFlags(nil)`
`klog.Flush()`

It should have these function call, even not sure but looks bug in there implementation

Reply:
>can you please open a issue in external-resizer to add support to it?

Sure, but hope that can be taken as a follow up in Rook, once their community has it fixed.

Reply:
I hope I opened it correct place https://github.com/kubernetes-csi/external-resizer/issues/411